lighthouse-core/fraggle-rock/gather/timespan-runner.js

```diff
@@ -22,6 +23,8 @@ const {getBaseArtifacts, finalizeArtifacts} = require('./base-artifacts.js');
  */
 async function startTimespan(options) {
   const {configContext = {}} = options;
+  log.setLevel(configContext.logLevel || 'error');
```

Comment:
Could move this into `initializeConfig` but it doesn't exactly fit. Running 3 different times isn't too big a deal anyway.